terraform-iac/prd/app/main.tf

```diff
@@ -20,8 +20,21 @@ module "app" {
   source    = "../../modules/app/"
   env       = "prd"
   image_tag = var.image_tag
+  codedeploy_termination_wait_time = 15
```

Comment:
I thought we wanted this shorter than 15 minutes

Reply:
0 in dev. 15 in prd